### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Atos20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,126 +1,64 @@
...
+## Project Status
+###
+The functionality to use the site is complete, however the site has not been tested thoroughly for user generated errors. 
+
+## Webite Screengrabs
+
+Generating random images:
+![image](https://user-images.githubusercontent.com/56229864/82391118-bb843200-99fd-11ea-904d-d58af18cf333.png)
+
+Saving and deleting saved images:
```
Comment (GreysonElkins) [inline README.md:16]: We gotta figure out how to do line breaks on these images

File: README.md [hunk 2]
```diff
@@ -1,126 +1,64 @@
...
+ROMCOM was built using using Atom, git workflow, and HTML and CSS provided by Turing.
+
+## Project Status
+###
+The functionality to use the site is complete, however the site has not been tested thoroughly for user generated errors. 
+
+## Webite Screengrabs
+
+Generating random images:
+![image](https://user-images.githubusercontent.com/56229864/82391118-bb843200-99fd-11ea-904d-d58af18cf333.png)
```
Comment (GreysonElkins) [inline README.md]: OH no our gifs are pngs now!

**Review by GreysonElkins — CHANGES_REQUESTED** [review]
Let's fix these images 👷‍♂️

File: README.md (hunk 1 above)
Comment (Atos20) [reply README.md:16]: DONE!

**Review by GreysonElkins — APPROVED** [review]
NICE!

**Pull request merged**